offerzen/global/employers/employers-signup-member-alerts@1.0.js

```diff
@@ -0,0 +1,45 @@
+!(function () {
+  window.$loaded(function (window, document, $, undefined) {
+    const params = new URLSearchParams(window.location.search);
+    $('.js-member-alert-message').val('');
+
+    if (params.has('member')) {
+      const alert = params.get('member');
+      //show the relevant message based on alert
+      switch (alert) {
+        case 'invalid_token':
+          $('.js-member-alert-message').text(
```

Comment:
Small thing, but I think we can move this into 1 line. Reads a little bit strange if it's split onto multiple

Reply:
moving it :D